netwulf/interactive.py

```diff
@@ -184,6 +194,30 @@ def visualize(network,
     filepath = os.path.join(web_dir, filename)
     configpath = os.path.join(web_dir, configname)
 
+    network = network.copy()
```

Comment:
Could you wrap the whole thing in 

```
if edge_weight != 'weight' or node_color is None
```

or is that not possible for some reason?